app/enterprise/2.1.x/secure-admin-api.md

```diff
@@ -115,6 +119,105 @@ From here, simply apply desired Kong-specific security controls (such as
 [IP restrictions][ip-restriction], or [access control lists][acl]) as you would
 normally to any other Kong API.
 
+If you are using docker to host Kong, you can accomplish a similar task via a declarative configuration such as this one:
+
+``` yaml
+_format_version: "1.1"
+
+services:
+- name: admin-api
+  url: http://127.0.0.1:8001
+  routes:
+    - paths:
+      - /admin-api
+  plugins:
+  - name: key-auth
+
+consumers:
+- username: admin
+  keyauth_credentials:
+  - key: secret
+```
+
+Under this configuration, the Admin API will be available via the `/admin-api` but only for requests accopanied with `?apikey=secret` query
+parameters.
+
+Assuming that the file above is stored in `$(pwd)/kong.yml`, a dbless Kong can use it as it starts like this:
```

Comment:
```suggestion
Assuming that the file above is stored in `$(pwd)/kong.yml`, a DB-less {{site.ee_product_name}} can use it as it starts like this:
```

For consistency https://docs.konghq.com/hub/plugins/compatibility/. I know just saying "Kong" was convenient but we should follow proper use of variables.

Reply:
I didn't know we were using those environment variables now. I have changed them (`site.ee_product_name` in Enterprise pages and `site.ce_product_name` in Community pages)